Lib/test/test_enum.py

```diff
@@ -1858,6 +1858,13 @@ class Decision2(MyEnum):
             REVERT_ALL = "REVERT_ALL"
             RETRY = "RETRY"
 
+    def test_compile(self):
```

Comment:
Please add a short comment mentioning bpo-35717 to explain the rationale of this test.

Reply:
Decorate the test with `cpython_only`. And I think it needs better name. It does not have particular relation with compiling. Maybe "test_empty_globals"?

Reply:
Hmm, no, decorating with `cpython_only` is not needed. The test should pass on other implementations even if they do not have `_getframe()`.

Reply:
Add the comment @vstinner suggested and you're good to go.

Reply:
Thanks, I added the comment